scripts/upgrade-v1-test/2_proposal_upgrade_v1.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# READ ME
+# This script is a test script to check basic operation of upgrade-v1.
+# Note: It is intended to be run in a clean environment.
+# Please be careful not to run it in a production environment or 
+# in an environment where ununifid has already been set up.
+set -e
+
+ununifid tx gov submit-proposal software-upgrade upgrade_v1 \
+--title upgrade-test-v1 \
+--description upgrade \
+--upgrade-info '{"binaries":{"linux/amd64":"https://github.com/hikaruNagamine/shared/releases/download/v1/ununifid_v1?checksum=md5:d8852a87392511f8b31bfadcb35a536f"}}' \
```

Comment:
In test, I think it's better to not define the binary location and checksum because we often change the source.
So let's remove this line.

And, you can include `deposit` in this tx by adding the following flag,
`--deposit 20000000000uguu`